api/src/resolvers/campaigns/swollen-sunday/swollen-sunday-campaign-cypher.ts

```diff
@@ -50,12 +50,16 @@ CALL apoc.cypher.run('
 
 WITH avg(value.bussing.attendance) as averageBacentaAttendance, bacenta
 WITH sum(averageBacentaAttendance) as averageCouncilBussing
-RETURN round(toFloat(averageCouncilBussing)) as averageCouncilBussing
+RETURN toFloat(averageCouncilBussing) as averageCouncilBussing
 `
 
 export const shareBacentaTargetsCypher = `
 MATCH (this:Council {id:$councilId})
 MATCH (this)-[:HAS]->(:Constituency)-[:HAS]->(bacenta:Active:Bacenta)
+OPTIONAL MATCH (bacenta)-[:HAS_HISTORY]->(:ServiceLog)-[:HAS_TARGET]->(oldTarget:Target {date:date($swellDate)})
+DETACH DELETE oldTarget
```

Comment:
What does this line do? @dabick14

Reply:
If you set the targets the first time for a swell date, when you come back and want to change the target, this deletes all the bacenta targets related to that date.